### ISSUE TITLE
send transaction before sync - it's stuck for a while now

### ISSUE THREAD

**Issue by benammann** [vergecurrency/VERGE#466]
## Prerequisites

* [x] Are you running the latest version?

## User Story:

* "I'm submitting a ... "
  - [ ] bug report
  - [ ] feature request
  - [x] support request 
  - [ ] translation request 

## Current Behavior

I sent the transaction before the wallet finished syncing. I've read that will be no problem unless my inputs don't change.

Status: 0/unconfirmed
Date: 12.01.2018 18:03
To: cryptopia DDtjbCEYRxytDiZvUJ996mMuZmK3WCFsyq
Debit: -15,309.12 XVG
Transaction fee: -0.502612 XVG
Net amount: -15,309.622612 XVG
Transaction ID: cda39de579814bc664817c646b9df1c344b467e31f95e9b71b8fb535c3bb19bf

since the transaction did not show up for 3 hours, i tried to do some debugging

i went to `help -> console` and ran

`checkwallet`
`repairwallet`

After executing those commands, my volume showed up again. I was a little bit confused and made a test transaction

Status: 487 confirmations
Date: 12.01.2018 20:55
To: cryptopia DDtjbCEYRxytDiZvUJ996mMuZmK3WCFsyq
Debit: -10.00 XVG
Transaction fee: -0.127437 XVG
Net amount: -10.127437 XVG
Transaction ID: d16fe3d27902a258ce7130b352a78f2f60dd6513f06da39d7efcad1ba5fa598d

By sending the Transaction the wallet showed my an alert dialog `Transaction Rejected ...` it still went throw but definitely used one of the first transactions inputs.

I also tried to import the private key from one of the input addresses of the first transaction into electrum-xvg and it shows me the following

![xvg](https://user-images.githubusercontent.com/24310981/34900646-1733f054-f802-11e7-9316-bd3bba91df4e.PNG)

I'm really scared that I've just lost my coins.

## Your Environment

Client name: VVraith
Client version: v4.0.1.0-unk-beta
Using OpenSSL version: OpenSSL 1.0.2k  26 Jan 2017
Build date Jan 8 2018, 20:42:15

* OS:
    - [x] I'm using Windows

**Comment by Jasonlu668** [comment]
Maybe having a sign to tell users if it's synced or not will be better. First time use the verge wallet. I was confused if it's synced or not after it was actually synced. Because users do not stare at the wallet console and watching the whole sync process.

**Comment by benammann** [comment]
still unconfirmed and not showing up in the blockchain.

**Comment by DasSmeagle** [comment]
Have the same problem. My transaction is visible in blockchain explorer, but not confirmation. What can I do? I added 1 XVG fee.

https://verge-blockchain.info/tx/f060642c1cce1b1296a127e19a49fc22215b9f73aafaca31843123adf33b022b

I did a test adding 2 XVG fee - wallet told my "unusually high fee" - I accepted. And that one went through in minutes.

https://verge-blockchain.info/tx/79e48d4a5014ceb734a62d60c6580ff0682ae3a32bd3a91900148363f7ff4f43

So it seems, the 1 XVG which is added automatically is not enough? But how can I fix the stuck transaction? There needs to be a feature to cancel/resend or up the fee for stuck transactions...

Or if ther already is a feature like that, I couldn't find it, googling for ~1 hour ;)

Thanks,
- Oliver
